index.html

```diff
@@ -1,51 +1,261 @@
+      </p>
+    </div>
+    <div id="action">
+      <h2>Contact your elected officials</h2>
+      <ul>
+        <li>
+          <b>Call your state legislators</b> and tell them not to accept any
+          deal other than congestion pricing as already passed.
+          <a
+            href="https://docs.google.com/document/d/1SFr7qAbSin2nyo_Mn8C1vGhS0-bDxYX4jZNxGSOs2R8/edit?eid=5116"
+            >Open Plans created a helpful script</a
+          >
+          that also details how to find your Senator and Assemblymember's
+          contact info. (If that page doesn't work, you can also use the
+          <a href="https://www.nysenate.gov/find-my-senator">Senate</a> and
+          <a href="https://www.assembly.state.ny.us/mem/search/">Assembly</a>'s
+          lookup pages.) If you can't get through by phone, try
+          <a href="https://faxzero.com/">sending a fax</a>!
+        </li>
+        <li>
+          <b>Call the governor's office</b> at 518-474-8390. If you can, try to
+          speak with someone (option 2) - but expect hold times to be long! If
+          you aren't sure what to say,
+          <a href="https://action.ridersalliance.org/emergency-call-6524/"
+            >use this nifty tool from Riders Alliance</a
+          >
+          to get talking points.
+        </li>
+        <li>
+          <b
+            ><a href="https://bit.ly/HochulHatesTheMTA"
+              >Send a letter to the governor, the MTA board, and your
+              legislators</a
+            ></b
+          >, courtesy NYC-DSA Ecosocialists.
+        </li>
+      </ul>
+      <h2>Upcoming events</h2>
+      <ul>
+        <li>
+          Friday, June 7, meet at 7 AM in downtown Manhattan:
+          <a href="https://action.ridersalliance.org/emergency-lobby-day-6724/"
+            >Bus to Albany to rally/lobby with Riders Alliance</a
+          >
+        </li>
+        <li>
+          Saturday, June 8, 3 PM, Times Square:
+          <a
+            href="https://www.reddit.com/r/MicromobilityNYC/comments/1d9phu7/enough_is_enough_this_saturday_lets_fight_back/"
+            >Rally with /r/MicromobilityNYC</a
+          >
+        </li>
+        <li>
+          Sunday, June 9, 12-1 PM, Broadway Junction:
+          <a
+            href="https://www.ridersalliance.org/events/emergency-rally-broadway-junction"
+            >Rally with Riders Alliance</a
+          >
+        </li>
+      </ul>
+    </div>
+    <div id="map"></div>
+  </body>
+  <script>
+    const MAPTILER_KEY = "KmFaB2G58GxhXYCZaw1u";
+    const map = new maplibregl.Map({
+      style: `https://api.maptiler.com/maps/basic-v2/style.json?key=${MAPTILER_KEY}`,
+      center: [-74.0066, 40.7135],
+      zoom: 8.5,
+      container: "map",
+      antialias: true,
+    });
+    map.on("load", () => {
+      map.addSource("senate", {
+        type: "geojson",
+        // Use a URL for the value for the `data` property.
+        data: "http://localhost:3000/map/updated_Senate.geojson",
```

Comment:
will have to update this URL as well for the published link

Reply:
I think it can just be relative